google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -379,7 +356,7 @@ def delete force: nil
         #
         # @!group Table
         #
-        def create_table table_id, name: nil, description: nil, fields: nil
+        def create_table table_id, name: nil, description: nil
```

Comment:
The approach of accepting Field objects pre-dates the yielding of the table updater object in this method. The yielded table object allows a user to configure the schema using schema API, which is a much easier way to define the schema and matches the API used by Rails migrations.

Accepting fields on a Schema object also adds lots of complexity to a frozen/immutable logic. Since we removed it from Schema I felt it was appropriate to remove it from this call as well.